chinglee-iot/FreeRTOS-Kernel
# Add RP2040 SMP porting

Add RP2040 SMP porting

Description
-----------


Test Steps
-----------
* Run RP2040 single/SMP main full demo
* Run WIN32 MSVC single main full demo

Related Issue
-----------


Performance Comparison Test
-----------
* SMP only PR. No performance difference.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
